lib/src/page_list_viewport_gestures.dart

```diff
@@ -242,6 +253,8 @@ class _PageListViewportGesturesState extends State<PageListViewportGestures> wit
   void _startMomentum() {
     PageListViewportLogs.pagesListGestures.fine(() => "Starting momentum...");
     final velocity = _panAndScaleVelocityTracker.velocity;
+    final momentumSimInitialVelocitySclar = _panAndScaleVelocityTracker.momentumSimInitialVelocitySclar;
```

Comment:
Please don't shorten words when we don't have to. "Sim" -> "Simulation", "Sclar" -> "Scalar".

Also, if you really do mean "velocity scalar", then you could use the word "speed" instead, which mens the same thing.

I'm also curious about introducing the term "momentum". Is the velocity up above not also based on momentum? If the tracker is all about momentum, then we shouldn't need to add that word onto individual properties. The same thing goes for "simulation". Unless I'm missing something, it seems like this property could simply be called `initialSpeed`.

Reply:
got it! From now on, we have the velocity, it's a vector. We can talk about it's magnitude, direction like that of a vector. Hopefully new comments reflect that.